utils.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs/promises');
```

Comment:
Your logic looks good but you should look at splitting this up in to a model & a controller. The model will handle the data with fs. 
utils.js may not be the most appropriate file name here.

Reply:
Should i use new model and controller. how do I name them.